install.sh

```diff
@@ -91,7 +91,7 @@ if command -v make > /dev/null 2>&1; then
 fi
 
 if command -v fpm > /dev/null 2>&1; then
-  FPM=`which fpm`
+  FPM=`$which fpm`
```

Comment:
What is the significance of this change?

Reply:
Whoops, that's a typo. I think I put $($REALPATH fpm)` there and realized that's not needed, but forgot to remove it completely.